aio/content/marketing/contributors.json

```
@@ -830,5 +830,13 @@
     "twitter": "ahasall",
     "website": "https://www.amadousall.com",
     "bio": "Amadou is a Frontend Software Engineer from Senegal based in France. He currently works at Air France where he helps developers build better Angular applications. Passionate about web technologies, Amadou is an international speaker, a technical writer, and a Google Developer Expert in Angular."
+  },
+  "carloscaballero": {
+    "name": "Carlos Caballero",
+    "picture": "carloscaballero.jpg",
+    "twitter": "Carlillo",
+    "website": "http://carloscaballero.io",
+    "bio": "Carlos Caballero is a Computer Science Engineer and a PhD. in Computer Science from the University of Málaga. Master in Software Engineering and Artificial Intelligence. He is Full-time teacher in vocational training. He has written several teaching books and more than a dozen research articles in the first international journals.",
```

```suggestion
    "bio": "Carlos Caballero is a computer science engineer and a PhD in Computer Science from the University of Málaga. He earned his masters in Software Engineering and Artificial Intelligence. He is a full-time teacher in vocational training. He has written several teaching books and more than a dozen research articles in the first international journals.",
```

Regarding this sentence:
> He is a full-time teacher in vocational training.
Could you clarify what you mean with vocational training? In the US there's a difference between vocational education (trade- or craft-based, technicians) and professional (law, the sciences, math, humanities - all areas in which one might pursue a PhD and work in academia).

And by "first international journals", would this be the same as peer-reviewed journals?

---

Hi @kapunahelewong !

Thank you very much for your suggestions, I will make the change with the one proposed by @ajitsinghkaler .

Regarding your questions, I try to answer them.

I think the training that I teach is equivalent in the US with "Associate Degree", in Europe there is no consensus and normally when I travel to other countries of the European Union we usually indicate it as Vocational Training Education.

Regarding the second question, I mean that they are publications of the first quartile in their category according to the JCR (Journal Citation Reports).

---

Oh, I see! Thank you for explaining, @caballerog. Apologies for the messed up formatting on my comment. 

Since you'll be making an extra commitment with your new changes, just remember to squash so that ultimately there's only one commit. I have [some notes](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash) on the process that you may find helpful. If you have any questions though, just let me know.